crates/db_views_actor/src/structs.rs

```diff
@@ -23,6 +25,15 @@ pub struct CommunityBlockView {
   pub person: Person,
   pub community: Community,
 }
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[cfg_attr(feature = "full", derive(TS))]
+#[cfg_attr(feature = "full", ts(export))]
+/// An instance block by a user.
+pub struct InstanceBlockView {
+  pub person: Person,
+  pub instance: Instance,
+  pub site: Option<Site>,
```

Comment:
Just to show extra Site data, like a sidebar (in the case of lemmy instances?) I guess it could be helpful for front ends, but *maybe* not necessary.

Reply:
Its necessary if we want to show icons and names of blocked instances, like we are doing with blocked users and communities.